content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -363,6 +367,65 @@ will be terminated once the job backoff limit has been reached. This can make de
 from failed Jobs is not lost inadvertently.
 {{< /note >}}
 
+### Backoff limit per index {#backoff-limit-per-index}
+
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+{{< note >}}
+You can only configure the backoff limit per index for a Job, if you
+have the `JobBackoffLimitPerIndex` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster. A per-index backoff limit is only available for indexed Jobs that
+use the `restartPolicy: Never`.
+{{< /note >}}
+
+When you run an [indexed](#completion-mode) Job, you can choose to handle retries
+for pod failures independently for each index. To do so, set the
+`.spec.backoffLimitPerIndex` to specify the the maximal number of pod failures
+per index.
+
+When the per-index backoff limit is exceeded for an index, the index is marked as failed in the
+`.status.failedIndexes` field. The succeeded indexes, those with a successfully
+executed pods, are recorded in the `.status.completedIndexes` field, as for
+regular Indexed jobs.
+
+Note that, a failing index does not interrupt execution of other indexes.
+You can additionally limit the maximal number of indexes marked failed by
+setting the `.spec.maxFailedIndexes` field. In that case,
+the the number of failed indexes is exceeded, then the entire Job is marked
+failed; marking that failure triggers termination of all running Pods for that Job.
+
+Once all indexes finish for a Job where you specified a per-index backoff limit,
+if at least one of those indexes did fail, the Kubernetes control plane
+marks the overall Job as failed. The Job gets marked as failed even if some,
+perhaps nearly all, of the indexes were processed successfully.
+
+Here is an example manifest for a Job that defines a `backoffLimitPerIndex`:
+
+{{< codenew file="/controllers/job-backoff-limit-per-index-example.yaml" >}}
+
+In the example above, the Job controller will allow for one restart for each
+of the indexes. When the total number of failed indexes exceeds 5, then
+the entire Job is terminated.
+
+Once the job is finished, the the Job status will look as follows:
+
+```sh
+kubectl get -o yaml job job-backoff-limit-per-index-example
+```
+
+```yaml
+  status:
+    completedIndexes: 1,3,5,7,9
+    conditions:
+    - message: Job has failed indexes
+      reason: FailedIndexes
+      status: "True"
+      type: Failed
+    failed: 10            # 2 failed pods (1 retry) for each of 5 failed indexes
+    failedIndexes: 0,2,4,6,8
```

Comment:
could move the `failedIndexes` to come after `completedIndexes`. Slightly different from the actual output but more helpful I think, and anyway this isn't the whole object.